invenio_requests/services/requests/facets.py

```diff
@@ -15,13 +15,20 @@
     field="type",
     label=_("Type"),
     value_labels={
-        "community-submission": _("Draft review"),
-        "community-inclusion": _("Community inclusion"),
-        "community-invitation": _("Member invitation"),
+        # Access
         "guest-access-request": _("Guest access"),
         "user-access-request": _("User access"),
-        "community-manage-record": _("Community manage record"),
+        # Community record and draft submission
+        "community-inclusion": _("Community inclusion"),
+        "community-submission": _("Draft review"),
+        # Membership
         "community-membership-request": _("Membership request"),
+        "community-invitation": _("Member invitation"),
+        # Subcommunity
+        "subcommunity": _("Subcommunity"),
+        # Zenodo TODO define types in a way which is extensible by sites/modules
```

Comment:
I would:

1) Remove `Zenodo` from the comment

```suggestion
        # The following labels are instance-specific and ideally should not be listed here
```

2) log an issue in `invenio-requests` to make this list configurable per instance